Makefile

```diff
@@ -121,11 +121,14 @@ docker-build-microsoft-vhd:
 docker-wait-pachd:
 	etc/compile/wait.sh pachd_compile
 
-docker-build: docker-build-job-shim docker-build-pachd docker-wait-job-shim docker-wait-pachd
+docker-build: docker-build-job-shim docker-build-pachd docker-wait-job-shim docker-wait-pachd docker-build-netcat
```

Comment:
Well since its a requirement for `make test` how about I list it there (after `docker-build`) as a sub task?